### PR TITLE
menu: two _onKeyDown handler binds if block is inited on focus (close #1381)

### LINKED ISSUE #1381: menu: two _onKeyDown handler binds if block inited on focus
https://github.com/bem/bem-components/blob/v2/common.blocks/menu/menu.js#L34
https://github.com/bem/bem-components/blob/v2/common.blocks/menu/menu.js#L42
If `menu` block inited with tab focus `_onKeyDown` handler bound twice. It causes scrolling two items with one cursor keypress.

### PR DESCRIPTION
**Pull request by dfilatov**
cc @narqo @Guria

### CODE REVIEW

**Comment by Guria** [comment]
lgfm

File: common.blocks/menu/menu.js [hunk 1]
```diff
@@ -131,6 +112,23 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends menu.
         return null;
     },
 
+    /** @override **/
+    _onFocus : function() {
```
Comment (narqo) [inline common.blocks/menu/menu.js:116]: Does this handler still work if block had had focus before it was inited?

**Comment by narqo** [comment]
The code looks good. If there are no problems with initial focus – :snowflake:

File: common.blocks/menu/menu.js (hunk 1 above)
Comment (dfilatov) [inline common.blocks/menu/menu.js:116]: Good catch :+1: Seems that no.
I think I've fixed it by the second commit. Take a look.

**Comment by qfox** [comment]
:+1:

**Pull request merged**